e2e/utils/index.ts

```diff
@@ -161,9 +157,7 @@ export function runCreateWorkspace(
 
   const pm = getPackageManagerCommand({ packageManager });
 
-  let command = `${pm.createWorkspace} ${name} --cli=${
-    cli || currentCli()
```

Comment:
There is a `cli` variable that I don't see the change to remove it. I can see several e2e tests passing that option to `runCreateWorkspace`. It's not an issue because it's now unused, but it's something to cleanup.